packages/api/src/get-build.js

```diff
@@ -47,7 +47,23 @@ class GetBuildHandler {
 
         var metadata = await this.client.collection(global.headCollection).doc(firebaseEncode(repoid)).get();
 
-        res.send({ metadata: metadata.data(), builds: resp });
+        res.send({ metadata: metadata.data() || {}, builds: resp });
+      } catch (err) {
+        res.status(500).send({ error: err.message, stack: err.stack, name: err.name });
+      }
+    });
+
+    this.app.get('/api/repo/:orgname/:reponame/build/:buildid', async (req, res, next) => {
+      try {
+        const repoid = req.params.orgname + '/' + req.params.reponame;
+        const buildid = req.params.buildid;
+
+        const doc = await this.client.collection(global.headCollection).doc(firebaseEncode(repoid)).collection('builds').doc(firebaseEncode(buildid)).get();
```

Comment:
Should `'builds'` be a constant somewhere?

Reply:
Created an issue to address this, and rename all "build" to "run" 
https://github.com/GoogleCloudPlatform/flaky-service/issues/40